draft-ietf-tls-deprecate-obsolete-kex.md

```diff
@@ -245,7 +235,7 @@ Bleichenbacher's attack would affect all endpoints sharing the same RSA key (see
 
 Given these problems, this document updates {{!RFC9325}}, {{!RFC4346}}, {{!RFC5246}},
 {{!RFC4162}}, {{!RFC6347}}, {{!RFC5932}}, {{!RFC5288}}, {{!RFC6209}}, {{!RFC6367}},
-{{!RFC8422}}, {{!RFC5289}}, and {{!RFC5469}} to remediate the above problems.
+{{!RFC8422}}, {{!RFC5289}}, {{!RFC4785}}, {{!RFC4279}}, {{!RFC5487}}, {{!RFC6655}}, {{!RFC7509}} and {{!RFC5469}} to remediate the above problems.
```

Comment:
I'm not sure about RFC 7509... Could you please explain the rationale for referencing it? thanks!

Reply:
Yup Should be 7905  ChaCha20-Poly1305 Cipher Suites for Transport Layer Security